modules/clientutils.js

```diff
@@ -659,6 +644,49 @@
         };
 
         /**
+         * Makes selector by defined type XPath, Name or Label. Function has same result as selectXPath in Casper module for 
+         * XPath type - it makes XPath object. 
+         * Function also accepts name attribut of the form filed or can select element by its label text.
+         *
+         * @param  String selector Selector of defined type
+         * @param  String|null  type Type of selector, it can have these values:     
+         *         css - CSS3 selector - selector is returned trasparently
+         *         xpath - XPath selector - return XPath object    
+         *         name|names - select input of specific name, internally covert to CSS3 selector
+         *         label|labels - select input of specific label, internally covert to XPath selector. As selector is label's text used.
+         * @return String|Object          
+         */
+        this.makeSelector = function makeSelector(selector, type){
+            type = type || 'xpath'; // default type
+            var ret;
+
+            if (typeof selector === "object") { // selector object (CSS3 | XPath) could by passed
+                selector = selector.path;
+            }
+
+            switch (type) {
+                case 'css': // do nothing
+                    ret = selector;
+                    break;
+                case 'name': // convert to css 
+                case 'names': 
+                    ret = '[name="' + selector + '"]';
```

Comment:
If the only reason for the switch is the singular alias, you might as well add it in the object-style as seen in the existing code, i.e.

```js
var finders = {
...
labels: ...
};
finders.label = finders.labels;
```